WEB_SERVICE/WEB/_MEDIUM/HTML_INJECTION/description.md

```diff
@@ -0,0 +1,105 @@
+### HTML Injection
+
+HTML injection is a security vulnerability that arises when an input point in a web application can be manipulated by a user, enabling them to insert arbitrary HTML code into a susceptible web page. Exploiting this vulnerability can lead to various attacks, such as leaking a user's session cookies, which can then be exploited for more attacks. Furthermore, it allows the attacker to alter the content displayed to the victims, granting them the ability to modify the page to insert malicious code or deface the page with their own message.
```

Comment:
Didn't we want this for Mobile Application, the context should apply to Mobile App, hence the code below `Dart` `Kotlin` ...